acceptance/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+build/builder.sh make install
```

Comment:
something along the lines of:
localcluster.go:
var tempDir = flag.String("tmpdir", "", "the temp directory to use for short-lived data")
L168: binds := []string{os.ExpandEnv(path.Join(*tempDir, "/certs:/certs"))}

acceptance/run.sh:
go run acceptance/*.go --tmpdir=$(mktemp -d)

Or create a tmp dir in localcluster.go if the dir does not exist.